src/card/color.rs

```diff
@@ -21,7 +21,7 @@ pub enum Colour {
 /// the search builder as a [`ColourParam`].
 ///
 /// [`ColourParam`]: ../../card_searcher/enum.ColourParam.html
-#[derive(Serialize, Deserialize, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
```

Comment:
For the same reason as above I think this could make sense but I'm not sure the derived version would wield good results.